docker/install.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+#  Copyright (C) 2018 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+
+# Build the software and its dependencies
+source /opt/ros/kinetic/setup.bash
+cd ~/carma_ws
+rosdep install --from-paths src --ignore-src --rosdistro=kinetic -y
+catkin_make install
```

Comment:
We need to scrub the private sensitive launch files for can id's. I believe the launch files will be installed in this step and they may have can id's as parameters.

Reply:
Prior to publication, I agree. We'd probably just want to install a basic non-sensitive version and map our specific launch files in at run time, based on vehicle type.